app/controllers/films_controller.rb

```diff
@@ -1,16 +1,43 @@
 require 'pry'
+require 'open-uri'
 class FilmsController < ApplicationController
+  def add_movie
+    movie_id = params[:movie_id]
+      movie = Film.create(
+        rapid_id: params['movie_id'],
+        title: params['title'],
+        rating: params['rating'],
+        genre: params['genre'],
+        year: params['year']
+      )
+    if movie.save
```

Comment:
Add Else Part of if movie.save

S1 : Remove line no 10 and 11 
S2 : Apply Validation on genre and years
S3 : Save Record and show the validation error to the user
S4 : attached the screenshot here to the user.

for  example remove the year and then test the else part it should show the user validation error 
Film.rb# add validation on years

-- 
screenshot attach showing the error to the user

- gener is missing,
- year is missing